src/implementations/twig/components/accordion/accordion.story.js

```diff
@@ -43,8 +61,16 @@ export default {
   title: 'Components/Accordion',
 };
 
-export const Default = () => accordion(prepareAccordion(demoData));
-
+export const Default = (args) => accordion(prepareDataWithArgs(demoData, args));
+Default.argTypes = argTypes;
 Default.storyName = 'default';
-Default.parameters = { notes: { markdown: notes, json: demoData } };
-Default.decorators = [withKnobs, withCode, withNotes];
+Default.parameters = {
+  notes: {
+    markdown: notes,
+    json: demoData,
+  },
+  knobs: {
```

Comment:
it will, but then once the migration will be completed we can have a single ticket to simply remove this from each component that has been migrated to controls. I find it weird to have both panels in place with the knobs one empty, just that...

Reply:
I like the approach of disable-first and remove-when-done approach here